1. Случайная непрерывная величина A имеет равномерное распределение на промежутке 
. Найдите ее среднее значение и дисперсию.

In [1]:
def mean_and_variance(a,b):
    return f'На промежутке ({a};{b}]\nСреднее значение: М(А) = {(a+b)/2: .2f}\nДисперсия: D(A) = {((b-a)**2)/12: .2f}'
print(mean_and_variance(200,800))

На промежутке (200;800]
Среднее значение: М(А) =  500.00
Дисперсия: D(A) =  30000.00


Ответ: 30000.00

2. О случайной непрерывной равномерно распределенной величине B известно, что ее дисперсия равна 0.2. Можно ли найти правую границу величины B и ее среднее значение зная, что левая граница равна 0.5? Если да, найдите ее.

In [2]:
b=0.5+2.4**(1/2)
print(f'Левая граница распределения величины В - b = {b: .3f}\n'
      f'Среднее значение В на промежутке (0.5; {b: .3f}) M(B) = {(b+0.5)/2: .3f}'     
     )

Левая граница распределения величины В - b =  2.049
Среднее значение В на промежутке (0.5;  2.049) M(B) =  1.275


Ответ: 
Левая граница распределения: b = 2.049
Среднее значение: M(B) = 1.275

3. Непрерывная случайная величина X распределена нормально и задана плотностью распределения
f(x) = (1 / (4 * sqrt(2pi))) * exp((-(x+2)**2) / 32)
Найдите:
а). M(X)
б). D(X)
в). std(X) (среднее квадратичное отклонение)

Ответ: не выполнил

4. Рост взрослого населения города X имеет нормальное распределение. Причем, средний рост равен 174 см, а среднее квадратичное отклонение равно 8 см. Какова вероятность того, что случайным образом выбранный взрослый человек имеет рост:
а) больше 182 см
б) больше 190 см
в) от 166 см до 190 см
г) от 166 см до 182 см
д) от 158 см до 190 см
е) не выше 150 см или не ниже 190 см
ё) не выше 150 см или не ниже 198 см
ж) ниже 166 см.

In [3]:
def z_value(height):
    return (height-174)/8

In [4]:
from statistics import NormalDist

In [5]:
print(z_value(182))

1.0


In [6]:
NormalDist().cdf(z_value(182))

0.8413447460685429

In [7]:
P=1-NormalDist().cdf(z_value(182))
P

0.15865525393145707

In [8]:
z=z_value(190)
z

2.0

In [9]:
P=1-NormalDist().cdf(z)
P

0.02275013194817921

In [10]:
z1=z_value(166)
z1

-1.0

In [11]:
z2=z_value(190)
z2

2.0

In [13]:
P=NormalDist().cdf(z2)-NormalDist().cdf(z1)
P

0.8185946141203637

In [14]:
z1=z_value(166)
z1

-1.0

In [15]:
z2=z_value(182)
z2

1.0

In [16]:
P=NormalDist().cdf(z2)-NormalDist().cdf(z1)
P

0.6826894921370859

In [17]:
z1=z_value(158)
z1

-2.0

In [18]:
z2=z_value(190)
z2

2.0

In [19]:
P=NormalDist().cdf(z2)-NormalDist().cdf(z1)
P

0.9544997361036416

In [20]:
z1=z_value(150)
z1

-3.0

In [21]:
z2=z_value(190)
z2

2.0

In [22]:
P=NormalDist().cdf(z1)+(1-NormalDist().cdf(z2))
P

0.024100029979809312

In [23]:
z1=z_value(150)
z1

-3.0

In [24]:
z2=z_value(198)
z2

3.0

In [25]:
P=NormalDist().cdf(z1)+(1-NormalDist().cdf(z2))
P

0.002699796063260207

In [26]:
z=z_value(166)
z

-1.0

In [27]:
P=NormalDist().cdf(z)
P

0.15865525393145707

Ответ:
а) 0.1587
б) 0.02275
в) 0.8186
г) 0.6827
д) 0.9545
е) 0.0241
ё) 0.0027
ж) 0.1587

5. На сколько сигм (средних квадратичных отклонений) отклоняется рост человека, равный 190 см, от математического ожидания роста в популяции, в которой M(X) = 178 см и D(X) = 25 кв.см?

In [28]:
Z=(190-178)/25**(1/2)
Z

2.4

In [29]:
print(f'Рост человека, равный 190 см, отклоняется от математического ожидания роста в популяции, \n'
      f'в которой M(X) = 178 см и D(X) = 25 кв.см на Z = {Z} сигм.')

Рост человека, равный 190 см, отклоняется от математического ожидания роста в популяции, 
в которой M(X) = 178 см и D(X) = 25 кв.см на Z = 2.4 сигм.


Ответ: на 2.4 сигмы